src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService activeUserCheck = new UserService();
+        User activeUser = new User(email, password);
+
+        if (activeUserCheck.findByEmail(email) != null) {
+            activeUser = activeUserCheck.findByEmail(email);
```

Comment:
```suggestion
        if (userService.findByEmail(email) != null) {
```
this line means that user exists and user's email is the same as passed email, no need to check it once more

Reply:
you can also use & operator and simplify code even more